astroquery/esa/hubble/tests/test_esa_hubble.py

```diff
@@ -32,32 +41,35 @@ def get_dummy_tap_handler(self):
     def test_download_product(self):
         parameters = {'observation_id': "J6FL25S4Q",
                       'calibration_level': "RAW",
-                      'filename': 'file',
-                      'verbose': False}
-        dummyHandler = DummyHandler("download_product", parameters)
-        ehst = ESAHubbleClass(dummyHandler, self.get_dummy_tap_handler())
+                      'verbose': True}
+        ehst = ESAHubbleClass(self.get_dummy_tap_handler())
+        target_file = data_path('download_product')
+        ehst._request = MagicMock(return_value=target_file)
+        shutil.move = MagicMock(return_value=True)
```

Comment:
This line is why you are getting the test failures. Whatever you moved here needs to be moved back when you are done. Is there a reason you use the `MagicMock` class rather than astroquery's `MockResponse`?